react-challenge.html

```diff
@@ -6,64 +6,70 @@
 
 <!-- HINT: Check the console in the browser developer tools for errors. -->
 
-
-
 <!DOCTYPE html>
 <html lang="en" dir="ltr">
-  <head>
-    <meta charset="utf-8">
-    <script src="https://unpkg.com/react@16/umd/react.development.js" crossorigin></script>
-    <script src="https://unpkg.com/react-dom@16/umd/react-dom.development.js" crossorigin></script>
-    <script src="https://unpkg.com/@babel/standalone/babel.min.js"></script>
-    <title>Mood Meter</title>
-  </head>
-  <body>
-    <div id="react-container"></div>
-    <script type="text/babel">
-    // No need to change anything ABOVE this line ^
-
-
-
-      class App extends React.Component{
-        constructor(props){
-          super(props)
-          this.state = {
-            moods: ["😃", "😴", "😍", "🥺", "😡", "🤓", "🤪"],
-            currentMood: ""
-          }
-        }
-
-        handleMoodOptions = () => {
-          let randomIndex = Math.floor(Math.random() * this.state.moods.length )
-          this.setState({ currentMood: moods[randomIndex] })
-        }
-
-        render(){
-          return(
-
-              <h1>Welcome to the Mood Meter App</h1>
-
-              <h3>Mood Options:</h3>
-
-              <div style={{ fontSize: "40px", height: "50px"}}>
-                { this.state.currentMood }
-              </div>
-
-              <button onClick={ handleMoodOptions }>
-                Click Here For Next Mood option
-              </button>
-
-          )
-        }
-      }
-
-
-
-
-
-      // No need to change anything below this line
-      const domContainer = document.querySelector('#react-container')
-      ReactDOM.render(<App />, domContainer)
-    </script>
-  </body>
+	<head>
+		<meta charset="utf-8" />
+		<script
+			src="https://unpkg.com/react@16/umd/react.development.js"
+			crossorigin
+		></script>
+		<script
+			src="https://unpkg.com/react-dom@16/umd/react-dom.development.js"
+			crossorigin
+		></script>
+		<script src="https://unpkg.com/@babel/standalone/babel.min.js"></script>
+		<title>Mood Meter</title>
+	</head>
+	<body>
+		<div id="react-container"></div>
+		<script type="text/babel">
+						// No need to change anything ABOVE this line ^
+
+
+
+						  class App extends React.Component{
+						    constructor(props){
+						      super(props)
+						      this.state = {
+						        moods: ["😃", "😴", "😍", "🥺", "😡", "🤓", "🤪"],
+						        currentMood: ""
+						      }
+			            this.handleMoodOptions = this.handleMoodOptions.bind(bind)(this)
```

Comment:
The syntax here seems a little strange because there are two arguments being supplied separately. The error here can be fixed without calling `.bind()` because latest versions of React have figured out a way to get around the scoping issue it once use to require using the latest version of ES6. The real issue here is more an issue of referencing the method a particular thing--remember that we're working within a component class and there's a particular way you need to reference a classes properties or methods within the class.